text/0119-instance-options.md

```diff
@@ -0,0 +1,459 @@
+- Title: Instance options
+- Start Date: 2022-03-01
+
+# Instance options
+
+## 1. Summary
+
+The instance options let the users configure Meilisearch when launching the search engine using
+- environment variables
+- command-line options
+
+An example when setting an environment variable to change the folder path where the Meilisearch data are stored:
+
+```bash
+export MEILI_DB_PATH=./meilifiles
+./meilisearch
+```
+
+Same behavior using the command-line option:
+
+```bash
+./meilisearch --db-path ./meilifiles
+```
+
+## 2. Motivation
+
+When Meilisearch is launched, the default configuration may not meet the specific needs of users. Meilisearch exposes configurable options to allow users to fine-tune the behavior of the search engine.
+
+## 3. Functional Specification
+
+The users can configure Meilisearch when launching the search engine using
+- **environment variables**. Ex: `--db-path`
+- **command-line options or CLI options**. Ex: `MEILI_DB_PATH`
+
+There are 2 categories of CLI (command-line interface) options:
+- the ones that expect a value. Ex: `--db-path`.
+- the ones that does expect any value, called also "flags". Ex: `--no-analytics`. Their implicit values are booleans.
```

Comment:
Shouldn't this be `don’t expect values`?

Reply:
Indeed!